cmd/goProbe/goProbe.go

```diff
@@ -161,6 +163,7 @@ func main() {
 	}
 
 	// Open control socket
+	_ = os.RemoveAll(filepath.Join(dbpath, CONTROL_SOCKET))
```

Comment:
Error handling, even if it's a RemoveAll() ;-)

Reply:
Fine... Even if this is opportunistic anyhow. What we shouldn't forget is to change this into a proper console log once #8 is merged.

Reply:
+1 ! My concerns about the error logging is just born out of own fuckups, I recently traced a bug for two days just to figure out that I could have tackled it in 10 minutes had I just logged an error in an RemoveAll() statement ;-)